File: CodeChef/JuneChallenge2020/ChefandPriceControl/PRICECON.cpp
#include <iostream>
using namespace std;

int main() {
	// your code goes here
	//below 2 lines are for running code on your local machine 
	//Remove these 2 lines while using code online on codechef as they provide thier own input
	freopen("input.txt", "r", stdin);
	freopen("output.txt", "w", stdout);
	
	int t;
	cin>>t;
	for(int i=0;i<t;i++)
	{
	    int n;
	    int k;
	    cin>>n;
	    cin>>k;
	    //vector<int> arr;
	    long long loss=0;
	    for(int j=0;j<n;j++)
	    {
	        long long no;
	        cin>>no;
	        if(no>k)
	            loss=loss+(no-k);
	            
	    }
	    cout<<loss<<endl;
	}
	return 0;
}

File: CodeChef/JuneChallenge2020/ChefandPriceControl/PRICECON.py
import sys 
# cook your dish here
#	below 2 lines are for running code on your local machine 
#	Remove these 2 lines while using code online on codechef as they provide thier own input
sys.stdin = open("input.txt",'r')
sys.stdout = open("output.txt",'w')

t =int(input())
for j in range(t):
    n,k = map(int,input().split())
    #n,q = map(int,input().split())
    arr = list(map(int,input().split()))
    loss=0
    for i in range(len(arr)):
        if arr[i]>k:
            loss=loss+(arr[i]-k)
    print(loss)
            
    
    
